src/doc/unstable-book/src/compiler-flags/print-crate-root-lint-levels.md

```diff
@@ -0,0 +1,23 @@
+# `print=crate-root-lint-levels`
+
+The tracking issue for this feature is: [#139180](https://github.com/rust-lang/rust/issues/139180).
+
+------------------------
+
+This option of the `--print` flag print the list of lints with print out all the lints and their associated levels (`allow`, `warn`, `deny`, `forbid`) based on the regular Rust rules at crate root, that is *(roughly)*:
+ - command line args (`-W`, `-A`, ...)
+ - crate root attributes (`#![allow]`, `#![warn]`, ...)
+ - *the special `warnings` lint group*
+ - the default lint level
```

Comment:
Question: is this print request supposed to account for `--force-warn` or `--cap-lints`?

Reply:
`--force-warn` is in the same category as `-W`, `-A` and `-D` so I would say yes.

`--cap-lints` is less obvious but since the goal is to know the lint level at crate root, I think it make sense to take it into account.

I have added tests for them and adjusted the documentation to mention them.